core/src/main/resources/defaults/blueprints/hdp30-shared-services-v2.bp

```diff
@@ -8,15 +8,24 @@
       "stack_name": "HDP",
       "stack_version": "3.0"
     },
-  "settings": [
-    {
-      "recovery_settings": []
-    },
-    {
-      "component_settings": []
-    }
+    "settings": [
+      {
+        "recovery_settings": []
+      },
+      {
+        "component_settings": []
+      }
     ],
     "configurations": [
+      {
+        "logfeeder-properties": {
```

Comment:
Nit: Could you please move these settings to just before the SmartSense settings. Trying to keep the core services further up in the configs.

Reply:
Fixed.